core/src/builder.ts

```diff
@@ -135,6 +136,17 @@ function Builder<Model>({
                   break;
               }
 
+              // TODO: Super hack to try some things out
+              if (type === 'rest') {
```

Comment:
This would go away once all builders exist in two variations, right?

Reply:
That is the idea, yes.